main.go

```diff
@@ -116,6 +119,21 @@ type KeepAliveConfig struct {
 	CountMax uint
 }
 
+func setupSshAgent(socket string) ssh.AuthMethod {
```

Comment:
Should be named `setupSSHAgent` according to typical Go style.

Reply:
GET